recipes/ngmix/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ngmix" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/esheldon/ngmix.git
+  git_rev: v{{ version }}
+  sha256: 17c8f5a9ead56096dd04e09563cd706ab1ffce4e87babadbbda131c5a38f8c03
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - numba <0.41.0a|>=0.42.0  # bug in numba 0.41* series
+    - scipy
+    - galsim
+    - emcee
+
+test:
+  imports:
+    - ngmix
+
+about:
+  home: https://github.com/esheldon/ngmix
+  license: GPL-2
+  license_family: GPL
+  license_file: gpl.txt
+  summary: 'Gaussian mixtures and image processing implemented in python and C'
+
+extra:
+  recipe-maintainers:
+    - beckermr
+    - esheldon
```

Comment:
@esheldon please confirm if you would like to help maintain this or not?

Reply:
I confirm

Reply:
I believe 0.42 also has the bug

Reply:
Ohhhh really. Mmk i'll bump to 0.43 then